activation/interface.go

```diff
@@ -24,4 +24,6 @@ type layerClock interface {
 	AwaitLayer(layerID types.LayerID) chan struct{}
 	GetCurrentLayer() types.LayerID
 	LayerToTime(types.LayerID) time.Time
+	// Convert Duration to number of layers
+	DurationToLayers(time.Duration) uint32
```

Comment:
I don't think a Clock should do Layer arithmetic, see also my comment further below